mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -366,7 +366,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5660 setSneaking (Z)V
 		ARG 1 sneaking
 	METHOD method_5661 getArmorItems ()Ljava/lang/Iterable;
-	METHOD method_5662 saveToTag (Lnet/minecraft/class_2487;)Z
+	METHOD method_5662 saveToNbt (Lnet/minecraft/class_2487;)Z
```

Comment:
I think this and `saveSelfToNbt`/`method_5786` need a closer look sometime later. These names are a bit weird, and a later PR really needs to rename them to be more descriptive.

- `saveToNbt` does `this.hasVehicle() ? false : this.saveSelfToNbt(tag)`
- `saveSelfToNbt` checks if there is a removal reason that should save, then puts the entity ID in a compound tag and calls `writeNbt` afterward.